### PR TITLE
Revert frac-lines to borders

### PR DESCRIPTION
**Pull request by ronkok**
This PR reverts the rendering of frac-lines from SVG paths back to span borders. This solves the thick grey bar reported by @mbourne in issue #1173.

The result is a frac-line similar to KaTeX v0.9.0-alpha1. The frac-lines are span borders and the CSS contains a `min-height: 1px;` rule to keep those borders visible.

There is one difference between this PR and v0.9.0-alpha1. The earlier work contained a second CSS rule:
```
@media screen and (-webkit-min-device-pixel-ratio: 2), screen and (min-resolution: 192dpi) {
      .katex .mfrac .fracline {
            min-height:0.5px;
    }
}
```
That second CSS rule did not help much. Instead, it caused some lines to disappear.  So this PR omits that second CSS rule.

In the thread to issue #1173, people tested the rendering from the proposed approach and reported that frac-lines disappeared only at sizes that were already too small to read.

### CODE REVIEW

File: src/environments/array.js [hunk 1]
```diff
@@ -202,8 +201,9 @@ const htmlBuilder = function(group, options) {
             }
 
             if (colDescr.separator === "|") {
-                const separator = stretchy.ruleSpan("vertical-separator", 0.05,
-                    options);
+                const separator = buildCommon.makeSpan(
+                    ["vertical-separator"], [], options
+            );
```
Comment (ylemkimon) [inline src/environments/array.js]: The indentation of this line seems to be wrong.

Comment (ronkok) [reply src/environments/array.js]: Indentation fixed. Thanks for the note.

**Comment by kevinbarabash** [comment]
@ronkok sorry for taking so long to look at this.  I'm generating the screenshots right now and will send you PR shortly.  I'd like to get this in before the next release.  Thank you for reaching out to the community to test this.

**Comment by kevinbarabash** [comment]
@ylemkimon thanks for updating that last Chrome screenshot.

**Pull request merged**